src/test/java/net/openhft/chronicle/queue/impl/single/IndexingLastSequenceNumberTest.java

```diff
@@ -27,7 +27,7 @@ void singleCycleOneEntryApproximateLookup() throws StreamCorruptedException {
         assertEquals(0, linearScanByPositionCountStart);
         long lastSequenceNumber = indexing.lastSequenceNumber(appender, true);
         assertEquals(0, lastSequenceNumber);
-        assertEquals(0, indexing.linearScanByPositionCount());
+        assertEquals(1, indexing.linearScanByPositionCount());
```

Comment:
we now always do a linear scan, but if we have consistent writePosition/sequence headers it should scan at most 2 non-metadata excerpts

Reply:
Perhaps we change this to assert that we don't do the scan-from-start?